### ISSUE TITLE
Merge into website? 

### ISSUE THREAD

**Issue by domoritz** [uwdb/uwdb.github.io#1]
Do you want to merge this into https://github.com/uwdb/website?

**Comment by akcheung** [comment]
What is the current organization of http://db.cs.washington.edu/projects.html ? Seems like each project points to its own domain rather than being hosted under uwdb/website? Maybe I didn't get your suggestion.

**Comment by domoritz** [comment]
>  Seems like each project points to its own domain rather than being hosted under uwdb/website

Not really. Most projects have their own directory in `projects/`. For example http://db.cs.washington.edu/projects/nids/ovmi.html. I think this repo should not exist since we could move quro into `uwdb/website/projects/quro`. Yes, myria, scidb and some others have their own websites but I think for smaller projects it is much easier to just put them inside `projects/`.

**Comment by akcheung** [comment]
Seems like those under `projects` are the older ones? PolyPEG, DopCert, PSLA are under `/`. We should bring this up in the next group meeting if we want to clean up the repo.

**Comment by akcheung** [comment]
Yes you are right (I thought you meant the code repo). @congy can you move the contents of this repo to  be under `https://github.com/uwdb/website/tree/gh-pages/projects`?

**Comment by domoritz** [comment]
Great. I think this will make things easier if all projects have a website under `uwdb/website/projects`. We should probably briefly talk about it in the group meeting.
